website/docs/guides/1.8.0-upgrade-guide.html.markdown

```diff
@@ -0,0 +1,34 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide"
+sidebar_current: "docs-mongodbatlas-guides-180-upgrade-guide"
+description: |-
+MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.8.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.8.0 has a number of new and exciting features and changes.
+
+**Major Enhancements:**
```

Comment:
Curious why change this from simply "New Features" as it's been for quite a few versions?

Reply:
updated to keep consistent with prior releases. wanted to highlight that we are summarizing a subset of new features, but rather not confuse existing users